packages/network-controller/src/NetworkController.ts

```diff
@@ -296,8 +530,12 @@ export class NetworkController extends BaseControllerV2<
 
   #providerProxy: ProviderProxy | undefined;
 
+  #provider: ProxyWithAccessibleTarget<Provider> | undefined;
```

Comment:
What is this doing? It looks like we set it but never use it? And its private so it couldn't used externally? cc @mcmire